docs/tutorials/e2e/boost/restrictData.md

```diff
@@ -150,23 +196,168 @@ Let´s see if Alice can see the Asset.
 Action (Alice): Execute a request using the following `curl` command:
 
 ```shell
-curl --location 'http://localhost/alice/management/v2/catalog/request' \
+curl --location 'http://dataconsumer-1-controlplane.tx.test/management/v2/catalog/request' \
 --header 'Content-Type: application/json' \
---header 'X-Api-Key: password' \
+--header 'X-Api-Key: TEST1' \
 --data-raw '{
     "@context": {},
     "protocol": "dataspace-protocol-http",
-    "counterPartyAddress": "http://bob-controlplane:8084/api/v1/dsp",
+    "counterPartyAddress": "http://dataprovider-controlplane.tx.test/api/v1/dsp",
     "querySpec": {
         "offset": 0,
         "limit": 100
     }
-}'
+}' | jq
+```
+
+The response doesnt include the cataolog entry for the asset/offer with id4
+
+```json
+{
+  "@id": "3b276870-5cc5-4546-b793-e2c8d39e1010",
+  "@type": "dcat:Catalog",
+  "dcat:dataset": [
+    {
+      "@id": "registry-asset",
+      "@type": "dcat:Dataset",
+      "odrl:hasPolicy": {
+        "@id": "ZGM3MjMyYTAtMDRjOC00MTVjLWI2NmQtOGJmNTQ1MWMyYmIy:cmVnaXN0cnktYXNzZXQ=:MjdiYWExMGItMTAzMC00MmY5LWI0YjMtMzJmY2UyODI5NThl",
+        "@type": "odrl:Set",
+        "odrl:permission": {
+          "odrl:target": "registry-asset",
+          "odrl:action": {
+            "odrl:type": "USE"
+          },
+          "odrl:constraint": {
+            "odrl:or": {
+              "odrl:leftOperand": "PURPOSE",
+              "odrl:operator": {
+                "@id": "odrl:eq"
+              },
+              "odrl:rightOperand": "ID 3.0 Trace"
+            }
+          }
+        },
+        "odrl:prohibition": [],
+        "odrl:obligation": [],
+        "odrl:target": "registry-asset"
+      },
+      "dcat:distribution": [
+        {
+          "@type": "dcat:Distribution",
+          "dct:format": {
+            "@id": "HttpProxy"
+          },
+          "dcat:accessService": "7b76c9c5-d7f9-42c1-8784-a2820a60bb0f"
+        },
+        {
+          "@type": "dcat:Distribution",
+          "dct:format": {
+            "@id": "AmazonS3"
+          },
+          "dcat:accessService": "7b76c9c5-d7f9-42c1-8784-a2820a60bb0f"
+        }
+      ],
+      "edc:type": "data.core.digitalTwinRegistry",
```

Comment:
again: it shouldn't look like this

Reply:
This asset was created by the seeedData function from the umbrella chart. What do you mean? This asset was created like this.